weberror/evalexception.py

```diff
@@ -254,13 +254,14 @@ def __call__(self, environ, start_response):
         environ['weberror.evalexception'] = environ['paste.evalexception'] = \
             self
         req = Request(environ)
-        if req.path_info.startswith('/'+self.debug_url_prefix):
+        req_path = req.path_info.split('/')[1:len(self.debug_url_prefix + 1)]
```

Comment:
This line is incorrect.  It attempts to add 1 to an array, and crashes.  The "+1" should be outside of the len() call.